modules/snooker/src/main/ts/ephox/snooker/resize/BarPositions.ts

```diff
@@ -17,14 +19,16 @@ export interface BarPositions<T> {
   readonly positions: (array: Optional<SugarElement>[], table: SugarElement) => Optional<T>[];
 }
 
-const rowInfo = (row: number, y: number): RowInfo => ({
+const rowInfo = (row: number, y: number, cell: SugarElement): RowInfo => ({
```

Comment:
Can we add the additional types here as well? eg `cell: SugarElement<HTMLTableCellElement>`